crates/swc/src/config/mod.rs

```diff
@@ -424,6 +428,9 @@ impl Options {
             40,
             NodeModulesResolver::new(TargetEnv::Node, Default::default(), true),
         );
+        let plugin_context = PluginContext {
+            source_file_name: source_file_name.clone(),
```

Comment:
This should use `base` instead.

Reply:
`source_file_name` is used to control sourcemap, and `base` is the filename.
It's named `base` because the parameter is introduced because of `paths` support, but I think it's a bad name.

Reply:
yes, I was confused by name actually.